tachyon/math/base/groups_unittest.cc

```diff
@@ -57,6 +57,39 @@ TEST(GroupsTest, DivOverMul) {
   static_cast<void>(c);
 }
 
+TEST(GroupsTest, InverseOverride) {
+  class IntInverse {
+   public:
+    IntInverse() : denominator_(0) {}
+    explicit IntInverse(int denominator) : denominator_(denominator) {}
+    IntInverse(const IntInverse& other) : denominator_(other.denominator_) {}
+
+    bool operator==(const IntInverse& other) const {
+      return denominator_ == other.denominator_;
+    }
+
+   private:
+    int denominator_;
+  };
```

Comment:
```suggestion
  class IntInverse {
   public:
    IntInverse() = default;
    explicit IntInverse(int denominator) : denominator_(denominator) {}
    IntInverse(const IntInverse& other) : denominator_(other.denominator_) {}

    bool operator==(const IntInverse& other) const {
      return denominator_ == other.denominator_;
    }

   private:
    int denominator_ = 0;
  };
```
How about like this as we used to do?